netbox_netprod_importer/__main__.py

```diff
@@ -97,6 +104,23 @@ def parse_args():
             if not isinstance(numeric_level, int):
                 raise ValueError('Invalid log level: %s' % args.verbose)
             logging.getLogger().setLevel(numeric_level)
+
+        args_h = vars(args)
+        args_h["creds"] = _get_creds(args)
+        print("Initializing importers…")
+        if args_h.get("devices"):
+            args_h["importers"] = parse_devices_yaml_def(
+                args.devices, args.creds
+            )
+        elif args_h.get("filter"):
+            args_h["importers"] = parse_filter_yaml_def(
+                args.filter, args.creds
+            )
+            # sys.exit(0)
+        else:
+            logger.error("Device file or filter file required")
+            sys.exit(3)
+
```

Comment:
You never seem to be using `args_h` after this point.

Reply:
Redid, wrong work with Namespace